daprovider/das/dasserver/dasserver.go

```diff
@@ -0,0 +1,191 @@
+package dasserver
```

Comment:
The 's' in 'das' stands for 'server' so the extra server is redundant.

Since you're changing a lot of names of packages in this PR already maybe take it as an opportunity to make the names clearer. da, das, dap; all confusing

Reply:
my bad, I misunderstood `das` to be "data-availability service" :) 
I'll figure out a better naming system!